server/src/main/java/com/saralein/server/connection/ConnectionHandler.java

```diff
@@ -7,6 +7,8 @@
 import com.saralein.server.response.Response;
 import com.saralein.server.response.ResponseSerializer;
 
```

Comment:
👾

Reply:
Yeah, I turned on the auto import feature in Intellij and it likes to add spaces between these and java packages.  Think I just figured out how to adjust it.